packages/client/src/createTRPCClient.ts

```diff
@@ -47,7 +47,8 @@ export type TRPCClient<TRouter extends AnyRouter> = DecoratedProcedureRecord<
   [untypedClientSymbol]: TRPCUntypedClient<TRouter>;
 };
 
-type ResolverDef = {
+/** @internal */
+export type ResolverDef = {
```

Comment:
remove @internal and smurf it as `TRPCResolverDef` ?

I've been wanting this type as well sometimes